docs/howto/contents.md

```diff
@@ -1,15 +1,28 @@
 # Azure Service Operator Documentation
 
-## Getting Started
+## Using Azure Service Operator
 
-1. [Prerequisites](prereqs.md):
-    Find the list of tools and dependencies needed to develop and deploy the Azure Service Operator here.
+4. [Build & deploy](deploy.md)
+    Build Azure Service Operator locally and deploy to a Kubernetes cluster.
+5. [Deploy with Helm](helmdeploy.md)
+   Deploy Azure Service Operator using a configurable Helm chart.
+6. [Provision a resource using Azure Service Operator](resourceprovision.md) Learn to use Azure Service Operator to provision different Azure resources.
+1. [Authentication using managed identities](managedidentity.md)
+   Use a managed identity for authentication of Azure Service Operator instead of a service principal.
+2. [Secret management](secrets.md) Learn how secrets are managed in Azure Service Operator and choose the best option for you.
+3. [How it works](controlflow.md) Learn the control flow behind deploying the operator and provision resources through the operator.
```

Comment:
renumber
```suggestion
1. [Build & deploy](deploy.md)
    Build Azure Service Operator locally and deploy to a Kubernetes cluster.
2. [Deploy with Helm](helmdeploy.md)
   Deploy Azure Service Operator using a configurable Helm chart.
3. [Provision a resource using Azure Service Operator](resourceprovision.md) Learn to use Azure Service Operator to provision different Azure resources.
4. [Authentication using managed identities](managedidentity.md)
   Use a managed identity for authentication of Azure Service Operator instead of a service principal.
5. [Secret management](secrets.md) Learn how secrets are managed in Azure Service Operator and choose the best option for you.
6. [How it works](controlflow.md) Learn the control flow behind deploying the operator and provision resources through the operator.
```